package.json

```diff
@@ -35,5 +34,7 @@
     "unbuild": "latest",
     "vitest": "^0.9.2"
   },
-  "packageManager": "pnpm@6.32.4"
+  "packageManager": "pnpm@6.32.4",
+  "repository": "github:unjs/defu",
```

Comment:
Provider is not needed either. Default is github when not provided. PS: Please use `npx sort-package-json`